### PR TITLE
Arcanna.ai v1.2 commands update

### PR DESCRIPTION
**Pull request by bperian**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Addition of new commands and overall update of the functionality of Arcanna.ai Integration.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 1]
```diff
@@ -13,342 +16,411 @@
...
+	def set_default_job_id(self, job_id):
+		self.default_job_id = job_id
+
+
+	def get_default_job_id(self):
+		return self.default_job_id
+
+	def test_arcanna(self):
+		url_suffix = 'api/v1/health/'
+		raw_response = requests.get(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify)
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:50]: Why are you using requests and not our [client class](https://xsoar.pan.dev/docs/integrations/code-conventions#client-class)?
It wrappes requests and handles everything for you

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 2]
```diff
@@ -13,342 +16,411 @@
...
+	def get_decision_set(self, job_id):
+		url_suffix = f"/api/v1/jobs/{job_id}/labels"
+		raw_response = requests.get(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify)
+		if raw_response.status_code not in [200, 422, 500]:
+			 raise Exception(f"Error HttpCode={raw_response.status_code} text={raw_response.text} Job ID :#{self.base_url}# #{url_suffix}# #{self.get_headers()}# #{job_id}# #{body}#")
+		return raw_response.json() 
+
+	def send_raw_event(self, job_id, severity, title, raw_body,id_value=None):
+		url_suffix = 'api/v1/events/'
+		if id_value is not None and id_value != "":
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:86]:
```suggestion
		if id_value and id_value != "":
```

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 3]
```diff
@@ -13,342 +16,411 @@
...
+		
+		return raw_response.json()
+
+	def map_to_arcanna_raw_event(self, job_id, raw, severity, title):
+		body = {
+			"job_id": int(job_id),
+			"title": title,
+			"raw_body": raw
+		}
+		if severity is not None:
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:104]:
```suggestion
		if severity:
```

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 4]
```diff
@@ -13,342 +16,411 @@
...
+		retry_count = 0
+		while(retry_count < retries):
+			url_suffix = f"api/v1/events/{job_id}/{event_id}"
+			full_url = f"{self.base_url}{url_suffix}"
+			raw_response = requests.get(url=full_url, headers=self.get_headers(), verify=self.verify)
+			if raw_response.status_code not in [200,422]:
+				raise Exception(f"Error HttpCode={raw_response.status_code} text={raw_response.text} Job ID :#{self.base_url}# #{url_suffix}#")
+			if raw_response.json()['status'] == 'pending_inference':
+				retry_count += 1
+				time.sleep(seconds_per_retry)
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:119]: Do not use sleep in your code.
Please see the following approach - https://xsoar.pan.dev/docs/tutorials/tut-design#command-design

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 5]
```diff
@@ -13,342 +16,411 @@
...
+	close_reason = args.get('close_reason', None)
+	pos = 0
+	for value in close_reasons:
+		if close_reason == value:
+			 if pos < len(labels):
+				 arcanna_label = labels[pos] 
+		pos += 1
+	if arcanna_label is None:
+		return_error(f"Error unknown label supplied.label={arcanna_label}")
+		raise Exception(f"Error in arcanna-send-event-feedback.Wrong label")
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:329]:
```suggestion
		raise Exception(f"Error in arcanna-send-event-feedback.Wrong label -  {arcanna_label}")
```

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 6]
```diff
@@ -13,342 +16,411 @@
...
-            raise BaseException("Arcanna: Error while parsing mapping fields")
-        demisto_closing_reason = parts[0].strip().replace("\"", "")
-        arcanna_label = parts[1].strip().replace("\"", "")
-        result[demisto_closing_reason] = arcanna_label
+	result = {}
+	pairs = mapping.split(",")
+	for pair in pairs:
+		parts = pair.split("=")
+		if len(parts) != 2:
+			return_error("Arcanna: Error while parsing mapping fields")
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:347]:
```suggestion
			Raise Exception("Arcanna: Error while parsing mapping fields")
```

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 7]
```diff
@@ -13,342 +16,411 @@
...
-    except Exception as e:
-        demisto.error(traceback.format_exc())  # print the traceback
-        return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
+	"""main function, parses params and runs command functions
+
+	:return:
+	:rtype:
+	"""
+
+	api_key = demisto.params().get('apikey')
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:363]: It's our convention now to use `credentials` for authentication.
https://xsoar.pan.dev/docs/integrations/code-conventions#credentials
We're strongly recommend to update this

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 8]
```diff
@@ -13,342 +16,411 @@
...
-        readable_output=readable_output,
-        outputs_prefix='Arcanna.Bulk',
-        outputs_key_field='status',
-        outputs=response
-    )
-
+def send_event_feedback(client: Client, close_reasons: list, args: Dict[str, Any]) -> CommandResults:
+	job_id = args.get("job_id", None)
+	if not job_id:
+		job_id = client.get_default_job_id()
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:314]: Why do you need this? 
Why not pass the param of default_job_id to the function directly, and do something like that
job_id = args.get("job_id", default_job_id)

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 9]
```diff
@@ -13,342 +16,411 @@
...
+
+	@staticmethod
+	def map_to_arcanna_label(arcanna_label, username):
+		body = {
+			"cortex_user": username,
+			"feedback": arcanna_label
+		}
+		return body
+
+	def send_bulk(self, job_id, events):
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:146]: It this used anywhere after all the changes?

**Review by Shellyber — CHANGES_REQUESTED** [review]
Thank you for your contribution.
1. It seems that you removed entirely the `send_bulk_events` command. is that intentional ? If so, this is breaking Backwards compatibility
2. Please add a release notes file with all the changes and the new commands. https://xsoar.pan.dev/docs/documentation/release-notes
3. Let's schedule a demo to review all the changes

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 9 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: removed

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 8 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: applied suggestion

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 6 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: applied suggestion

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 5 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: applied suggestion

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 4 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: removed sleep and retry mechanism as there's no support for retrying a step that has valid response ( _http_request has the retry mechanism but it's limited to status codes , not being able to use the remote response as a retry mechanism condition, at least from what I read in github for that class )

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 2 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: applied suggestion

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py (hunk 1 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py]: applied suggestion

File: Packs/Arcanna/pack_metadata.json [hunk 10]
```diff
@@ -2,13 +2,14 @@
     "name": "ArcannaAI",
     "description": "Siscale Arcanna.Ai Cognitive automation platform that provides AI assistance to IT & Cybersecurity teams",
     "support": "partner",
-    "currentVersion": "1.1.10",
+    "currentVersion": "1.2.10",
```
[the remark this answers is not in the document]
Comment (bperian) [reply Packs/Arcanna/pack_metadata.json]: applied suggestion

File: Packs/Arcanna/Integrations/ArcannaAI/README.md [hunk 11]
```diff
@@ -14,8 +14,6 @@ This integration was integrated and tested with version 1.2.2 of Arcanna.AI
     | Trust any certificate (not secure) |  | False |
     | Use system proxy settings |  | False |
     | Default Arcanna Job Id |  | False |
-    | A closing reason to Arcanna labels. | An array to map between closing reason and Arcanna labels | False |
-    | Alert closing reason | This field use to signal to Arcanna the status for closing an alert or marking feedback for Arcanna | False |
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/README.md:18]: Why you removed this?

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 12]
```diff
@@ -1,181 +1,521 @@
...
...qrz8/LQs/taPc0FwQXBBcEGyT4MLEojBBcEFwQXBBYnuwnEIBBfkUBwuSNCF5RBIwIVwCOQpUsmDUyABFzWnQFapcUE2XHCxRdYxuCAbLrjAIOu0nQsuMMg6rnPBBQZ5WlN7FywkZL2O9C4YPMkyDS5IcB3pXbCQkPXY2btg8CTL2NEFgyfx85DRBQMGWY2dg4uWoyBe2h8X3HiS1XgxuODTHLIaL0YXDBhkOV6MLhgwyHK8GF0wYJDleDG64AaDzKnl1wUDBpliPBc0EjLFeS7YVIm/pc4uaCTEbyOTC961E7+NTC7YSIjfRmYXNBLitZHZBRsJ6WNXLmgkpJoutXwXLYdCxncjCxdcYZDqYZ9dMHmSRp5dMHkSF3AhDefC1BlwwZ0nU2fIBZOn9qlTwi5YVSkXIRcUDMpF0AUFQ3PMpgsKhuZyYbddUDAoFyEXFAzKRdAFBYNlJOSC1+0sI0EXXHpSLkIuKBgqU8ueC16raozbdcF3GOp31LALy66qfEcNu2BXVT90hl3wgY72oXPDBZ1EexcJu6CTKB86t1zQSZR3kS0XdBLdXWTLBZ1E8y7ywgW3W0rSyHsuLO9JFHeRbRfiGDEUsHDyrgtGDL3DxUsXjBhqh4vXLvgUQ+XNxb4LZk+dM+eeC2ZPnTPnrgu+9lQ5c+67YClRymLPBUtJoTFyzQWvVrVtqAddAEMjiwMugKGQxREXwCgstY3jAhjqWBxzAQxdTeSwC2AoY3HUBTB0sTjsQlrOtIAYie0CGAWklfgueLuaex7HWbzjQhzfY2TNwkkaF2KZPgu/tjjngterCibOUy6kZcgofbQ45YKPPrPsIU5Su6CXZNhDrNzggoW18B5y1gV7SU5prNzlgvGz7GJx3oVYpoxyi8UFF0wZORQLd/rpnnfRLSbI+GgV5sKzveKC+bPIFnLdBa/SCrrJiuoCGUVtIRFdsLMWqCKKi04GNaMsFZFc0E2KmSsiu2A3+YwdxMV6nNFc9FegDBp/e19h4z3MiC76dtJAI/tSkcBFVzRa+skfoGht5OcY2wU07kdhbPyHmMDFSIOGcstMEb9SpHQxzBqG3TXxTmpcsqeXzsVog7qRqE4kNJHexdBTOhxUjohVoiNhkz+19C5mHR0PfFzw0IG4Q8S9LjwgfQw5mrY/L3v3Y/oP1Aw0t7bBHQQAAAAASUVORK5CYII=
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:8]: Please remove the image from the yml file. 
You can run demisto-sdk format and validate on the file to make sure it in the right format.
https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/format/README.md

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 13]
```diff
@@ -1,181 +1,521 @@
...
-- additionalinfo: URL of Arcanna API
-  defaultvalue: https://arcanna.ai
-  display: Server URL (e.g. https://<your arcanna ai api>)
+- display: Server URL (e.g. https://<your arcanna ai api>/ )
   name: url
-  required: true
+  defaultvalue: https://arcanna.ai/
   type: 0
-- additionalinfo: Api Key for Arcanna API
-  display: API Key
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:13]: You can deprecate and hide the old param to not break backward compatibility

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 14]
```diff
@@ -1,181 +1,521 @@
...
   type: 0
   required: false
-- additionalinfo: An array to map between closing reason and Arcanna labels
-  defaultvalue: '"False Positive"="drop_alert", "Resolved"="escalate_alert","Other"="escalate_alert","Duplicate"="escalate_alert"'
-  display: A closing reason to Arcanna labels.
-  name: feature_mapping
-  type: 12
-  required: false
-- additionalinfo: This field use to signal to Arcanna the status for closing an alert or marking feedback for Arcanna
-  defaultvalue: closeReason
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:37]: Why did you remove those param??

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 15]
```diff
@@ -1,181 +1,521 @@
...
+
+    # Disable insecure warnings
+    urllib3.disable_warnings()
+
+    ''' CLIENT CLASS '''
+
+
+
+
+class Client(BaseClient):
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:66]: It seems that you uploaded here the unified yml instead of the regular one.

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py [hunk 16]
```diff
@@ -150,205 +106,219 @@ def get_jobs(client: Client) -> CommandResults:
...
+	"""
 
-        client = Client(
-            api_key=api_key,
-            base_url=base_url,
-            verify=verify_certificate,
-            proxy=proxy,
-            default_job_id=default_job_id
-        )
+	api_key = demisto.params().get('credentials', {}).get('password')
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py:264]: You can do something like this to not break BC:
api_key = demisto.params().get('credentials', {}).get('password') or demisto.params().get('apikey')

**Review by Shellyber — CHANGES_REQUESTED** [review]
Nice work on getting it to work with the BaseClient
Few more notes:
1. It seems that you've uploaded the unified yml by mistake instead of the correct one.
2. You need to bump the pack version and add release notes and backward compatibility notes, since you seem to change the integration a lot (removed commands, removed params)
3. After the changes, lets schedule a demo.

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback_README.md
Comment (Shellyber) [inline Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback_README.md:1]: Why did you remove this?
Please revert this change

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml
Comment (Shellyber) [inline Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml:1]: Why did you remove this?
Please revert this change
If you don't need this playbook anymore - please just deprecate it but do not remove the file

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigation/playbook-Arcanna-Generic-Investigation_README.md [hunk 17]
```diff
@@ -29,4 +36,4 @@ There are no outputs for this playbook.
 
 ## Playbook Image
 ---
-![Arcanna-Generic-Investigation](https://user-images.githubusercontent.com/6702878/124025997-cdeede80-d9f9-11eb-8faa-e297c7cd3ca7.png)
+![Arcanna-Generic-Investigation](https://github.com/demisto/content/assets/24822739/96220ef5-a3af-4ff7-bd0f-29336f8e1e02)
```
Comment (Shellyber) [inline Packs/Arcanna/Playbooks/ArcannaGenericInvestigation/playbook-Arcanna-Generic-Investigation_README.md:39]: Is the image updated as well with the playbook changes?

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 18]
```diff
@@ -1,181 +1,203 @@
...
+      description: Arcanna generated unique event id.
     outputs:
     - contextPath: Arcanna.Event.event_id
-      description: Arcanna event id.
+      description: Arcanna event id
       type: String
     - contextPath: Arcanna.Event.ingest_timestamp
       description: Arcanna ingestion timestamp.
       type: String
-    - contextPath: Arcanna.Event.confidence_level
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:106]: So removing outputs from the yml also breaks backward compatibility. 
I would leave it as is and add the new values that return.

**Review by Shellyber — CHANGES_REQUESTED** [review]
Great Job!
Please see my recent notes :)
Let me know if you need any help with it.

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml (hunk 18 above)
Comment (bperian) [reply Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:106]: re-added

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml
Comment (bperian) [reply Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml:1]: reverted

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback_README.md
Comment (bperian) [reply Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback_README.md:1]: reverted

File: Packs/Arcanna/ReleaseNotes/1_1_11.md [hunk 19]
```diff
@@ -1,18 +0,0 @@
-
-#### Integrations
```
[the remark this answers is not in the document]
Comment (bperian) [reply Packs/Arcanna/ReleaseNotes/1_1_11.md]: reverted

File: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml
Comment (Shellyber) [reply Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml:1]: Don't forget to add the deprecation message as well to the release notes @bperian

File: Packs/Arcanna/ReleaseNotes/1_2_0.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
+#### Integrations
+
+##### Playbooks
```
Comment (Shellyber) [inline Packs/Arcanna/ReleaseNotes/1_2_0.md:3]: Please use the correct format with update-release-notes command in the SDK. 
The playbooks shouldn't be under the intergations

File: Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml [hunk 21]
```diff
@@ -1,181 +1,209 @@
-category: Data Enrichment & Threat Intelligence
```
Comment (Shellyber) [inline Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml:1]: Please revert this change

**Review by Shellyber — CHANGES_REQUESTED** [review]

**Comment by bperian** [comment]
Couldn't find the codes in the documentation for the remaining issues in CircleCI, suggestions are welcome

**Comment by bperian** [comment]
@Shellyber  I can't get the CICircle to pass entirely ( I only have one BC104 validation error left) . Any suggestions ? 

when I run `demisto-sdk validate  -i Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml -sv BC104` on my local branch it passes. I also double checked the code used for BC104, not sure what's wrong.

**Comment by Shellyber** [comment]
@bperian It fails because the command `arcanna-send-event-feedback` was removed from the yml. It may be accidentally removed from there. Need to re-add it

**Comment by Shellyber** [comment]
@bperian No problem :) 
Now just the pre-commit left

File: Packs/Arcanna/ReleaseNotes/1_2_0.md [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+##### Arcanna-Generic-Investigation
+- Added new playbook version of `Arcanna-Generic-Investigation` playbook, includes all available commands in current version
+
+##### Arcanna-Generic-Investigation-V2-With-Feedback
+- Deprecated. Use `ArcannaGenericInvestigation` instead.
+
+#### Scripts
+##### PrepareArcannaRawJson
+
+- Deprecated. Not used anymore.
```
Comment (Shellyber) [inline Packs/Arcanna/ReleaseNotes/1_2_0.md:44]:
```suggestion
- Deprecated. No available replacement.
```

File: Packs/Arcanna/ReleaseNotes/1_2_0.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+##### ArcannaFeedbackPostProcessing
+- Deprecated. Not used anymore.
```
Comment (Shellyber) [inline Packs/Arcanna/ReleaseNotes/1_2_0.md:46]:
```suggestion
- Deprecated. No available replacement.
```

File: Packs/Arcanna/Scripts/PrepareArcannaRawJson/PrepareArcannaRawJson.yml [hunk 24]
```diff
@@ -3,11 +3,11 @@ args:
   description: Input string to transform to JSON escaped object.
   name: input
   required: true
-comment: Loads a json from string input, and returns a json escaped result.
+comment: Deprecated. NOT_USED No available replacement.
```
Comment (Shellyber) [inline Packs/Arcanna/Scripts/PrepareArcannaRawJson/PrepareArcannaRawJson.yml:6]:
```suggestion
comment: Deprecated. No available replacement.
```

File: Packs/Arcanna/Scripts/ArcannaFeedbackPostProcessingScript/ArcannaFeedbackPostProcessingScript.yml [hunk 25]
```diff
@@ -11,9 +11,7 @@ args:
 - default: true
   description: aaa.
   name: closing_reason
-comment: |-
-  Arcanna.Ai post-processing script for sending feedback back to Arcanna about the closed incident. Additional modification might be required depending on each Cortex setup.
-  This script is intended to be used as a sample or in conjunction with the Arcanna-Generic-Investigation playbook.
+comment: Deprecated. NOT_USED No available replacement.
```
Comment (Shellyber) [inline Packs/Arcanna/Scripts/ArcannaFeedbackPostProcessingScript/ArcannaFeedbackPostProcessingScript.yml:14]:
```suggestion
comment: Deprecated. No available replacement.
```

**Comment by Shellyber** [comment]
@bperian The pre-commit step still fails. Please have a look and fix accordingly.

**Comment by bperian** [comment]
@Shellyber code coverage passing now.

**Comment by bperian** [comment]
@Shellyber could you please assist with this error ? Not sure what it refers to :

Could not parse content item from path: Packs/Arcanna/Playbooks/ArcannaGenericInvestigationV2WithFeedback/playbook-Arcanna-Generic-Investigation-V2-With-Feedback.yml: 3 validation errors for Playbook
id
  none is not an allowed value (type=type_error.none.not_allowed)
name
  none is not an allowed value (type=type_error.none.not_allowed)
display_name
  none is not an allowed value (type=type_error.none.not_allowed). Parser class: <demisto_sdk.commands.content_graph.parsers.playbook.PlaybookParser object at 0x7f9f17e278b0>
Could not parse content item from path: Packs/Arcanna/Playbooks/ArcannaGenericInvestigation/playbook-Arcanna-Generic-Investigation.yml: 3 validation errors for Playbook
id

**Comment by Shellyber** [comment]
@bperian yes, I'm checking why it fails on the build

**Comment by Shellyber** [comment]
@bperian I found the problem. 
The path of the playbooks is incorrect. We do not support folders under the `Playbooks` folder.
Please move all the files to be directly under the `Playbooks` folder and remove the playbooks folders you created there

**Comment by bperian** [comment]
CircleCI now complaints regarding the change done for precommit. 

`[2024-06-24 10:53:19,320] [ERROR] Packs/Arcanna/Playbooks/ArcannaGenericInvestigation/playbook-Arcanna-Generic-Investigation_README.md: [BA115] - The file Packs/Arcanna/Playbooks/ArcannaGenericInvestigation/playbook-Arcanna-Generic-Investigation_README.md cannot be deleted. Please restore the file.`

**Comment by Shellyber** [comment]
@bperian That's ok. I'll handle this internally

**Review by Shellyber — APPROVED** [review]

**Comment by bperian** [comment]
Thank you for the assistance !
